cloudimg/ms_azure.py

```diff
@@ -400,7 +400,10 @@ def upload_to_container(self, image_path, container_name, object_name,
                     container_client.container_name
                 )
             )
-            return
+            return self.get_object_by_name(
+                        container=container_name,
+                        name=object_name,
```

Comment:
Are we 100% sure that object which we found by `tags` will be 100% the same as will get by `object_name`

Reply:
Yes because we tag them after upload, shouldn't be any different. Also the `object_name` comes with the `.xz` already stripped so it's good to go.. But if you prefer I can return whatever is found in tags search for the safety sake